### PR TITLE
correct the etc symlink introduced in #71

### PR DESCRIPTION
**Pull request by dharmeshkakadia**

### CODE REVIEW

File: presto-yarn-package/src/main/slider/package/scripts/presto_server.py [hunk 1]
```diff
@@ -39,7 +39,7 @@ def start(self, env):
         env.set_params(params)
 
         self.configure()
-        os.symlink('{conf_dir}','{presto_root}')
+        os.symlink(format('{conf_dir}'),format('{presto_root}/etc'))
```
Comment (akshatnair) [inline presto-yarn-package/src/main/slider/package/scripts/presto_server.py]: use os.path.join?

**Review by akshatnair — APPROVED** [review]
looks good except a small comment

**Comment by dharmeshkakadia** [comment]
@akshatnair Does this look good ?

**Review by akshatnair — APPROVED** [review]

**Comment by dharmeshkakadia** [comment]
@akshatnair / @anusudarsan can you please help me merge this ?

**Comment by akshatnair** [comment]
Can you squash the commits, I will merge after that

**Comment by dharmeshkakadia** [comment]
Thanks @akshatnair. I have squashed the commits.

**Pull request merged**